package.json

```
@@ -82,6 +82,11 @@
                         "default": true,
                         "description": "Controls whether the editor should show diagnostics, such as warnings for unknown patterns."
                     },
+                    "hex-casting.showInternalNameHints": {
```

```suggestion
                    "hex-casting.inlayHints.internalName": {
```
Would prefer to follow the style of the Pylance and TypeScript extensions.

---

Also - if you don't mind, can you add a config for the macro inlay hints while we're at it? Consistency is good. That one can default to true.